go.mod

```diff
@@ -6,15 +6,17 @@ require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/fsnotify/fsnotify v1.6.0
 	github.com/go-logr/logr v1.2.4
+	github.com/hashicorp/go-multierror v1.1.1
```

Comment:
note https://github.com/kubevirt/kubevirt/issues/10283

Reply:
I think libraries will maintain MPL-2.0, only tools like terraform and vault will be under a Business Source License

Reply:
ack